File: 백준/Bronze/5176. 대회 자리/대회 자리.py
import sys

k = int(sys.stdin.readline())
for _ in range(k):
    p, m = map(int, sys.stdin.readline().split())
    seats = [0] * (m + 1)
    cnt = 0
    for _ in range(p):
        s = int(sys.stdin.readline())
        if seats[s] == 0:
            seats[s] = 1
        else:
            cnt += 1
    print(cnt)

File: 백준/Bronze/5176. 대회 자리/대회 자리.cc
#include <iostream>
using namespace std;

int main() {
	cin.tie(0)->sync_with_stdio(0);

	int k;
	cin >> k;
	while (k--) {
		int p, m;
		cin >> p >> m;
		int seats[501] = { 0, };
		int cnt = 0;
		for (int i = 0; i < p; i++) {
			int s;
			cin >> s;
			if (seats[s] == 0) seats[s] = 1;
			else cnt++;
		}
		cout << cnt << '\n';
	}
}